tests/unit/lms/services/assignment_test.py

```diff
@@ -239,55 +239,52 @@ def test_is_member(self, svc, db_session):
         assert svc.is_member(assignment, user.h_userid)
         assert not svc.is_member(assignment, other_user.h_userid)
 
-    def test_get_assignments(self, svc, db_session):
-        assert db_session.scalars(svc.get_assignments()).all()
-
-    def test_get_assignments_by_course(self, svc, db_session, assignment):
+    @pytest.mark.parametrize("instructor_h_userid", [True, False])
```

Comment:
This pattern tests all filterings indepently and combined. 

I reckon it makes for a better test. If we merge this I'll apply the same refactor to the user and course services.